guide/daily-spending-wallet/backup-and-recovery/manual-backup.md

```diff
@@ -21,64 +21,39 @@ Illustration sources:
-width = 1600
-height = 600
-layout = "full-width"
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual@2x.png"
+   mobile = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual-mobile.png"
+   mobileRetina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual-mobile@2x.png"
+   alt-text = "Backing up a recovery phrase chapter header image"
+   width = 1600
+   height = 600
+   layout = "full-width"
 %}
 
 # Manual backup
 
-## Backing up channel state
-
-A channel state backup is typically a file which includes information about the user's channel balances. This file can be downloaded onto the user's device and used to restore the Lightning channel state on another device. This could be a database of the most current channel balance information, or a static backup (as an absolute last resort).
-
-{% include tip/open.html color="red" icon="forbid" label="Manual channel state backups are for advanced users" %}
+The daily spending wallet described in this reference design is designed to make backup as easy as possible for the user. As a result, the default flow is one where the user opts-in to a cloud backup.
 
-Manual backups of Lightning channel state should only be performed with careful consideration. This is because the channel state backup may become outdated very quickly, and attempting to restore from an outdated backup could harm the user's funds. In the interest of adhering to the principles of [self-custody]({{ '/guide/getting-started/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/getting-started/principles/#transparency' | relative_url }}), you may choose to allow your user to perform a manual channel state backup. However, this is not recommended for the majority of users. If this option is available, keep it tucked away in a settings menu and prepend a warning that the option is only for advanced users.
+However, some users may not be comfortable with the idea of their private key on a cloud server. So in accordance with the design principles of [self-custody]({{ '/guide/getting-started/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/getting-started/principles/#transparency' | relative_url }}), this wallet also offers a manual backup option.
 
-{% include tip/close.html %}
+## A note about channel state
 
-## Backing up wallet recovery data
+The lightning channel state is subject to change frequently, particularly if the user truly uses the daily spending wallet every day as the name would imply. As a result, we can’t think about the channel state as being a one-time backup. Channel state must be backed-up everytime the state is updated.
```

Comment:
```suggestion
The lightning channel state will change frequently as the user makes and receives payments with this wallet. As a result, we can’t think about the channel state as being a one-time backup. Channel state must be backed-up every time the state is updated.
```
Also sometimes you use lightning, sometimes Lightning - - lets just go with lowercase throughout? @GBKS

Reply:
Lowercase lightning maximalism

Reply:
Done